server/views/components/sortable-table-header/macro.njk

```diff
@@ -0,0 +1,11 @@
+{#  -------------------------------------------------------------
```

Comment:
This (and the next file) are a move and refactor of `sortableTableHeader.njk` from the prisoner list page
I've made it generic so it can be used on both the Prisoner List page, and also the 3 new Session List pages 👍